td-agent/yum/almalinux-9-aarch64/from~

```diff
@@ -0,0 +1,18 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This file should be a backup file of editor? Not needed to included in this repo.